app/ts/App.tsx

```diff
@@ -356,7 +356,10 @@ export function App() {
 				<div class = 'form-group highlight'>
 					<h3>Fund AugurV2 Fork</h3>
 					<p>
-						Deposit REP to { <EtherScanAddress name = { 'REP Crowdsourcer' } address = { useComputed(() => getRepCrowdSourcerAddress()) }/> } that Micah can withdraw from once <b>{ bigintToDecimalStringWithUnknown(requiredBalance.deepValue, 18n, 2) } REP</b> is reached. Micah commits (via a gentleman's agreement) to use all of the REP to fund an Augur v2 fork. The REP will be lost. Please find Micah's more detailed explanation at <a href = '/blog.html' target = '_blank' rel = 'noopener noreferrer'>Go Fund Micah Announcement</a>.
+						Deposit REP to REP Crowdsourcer that Micah can withdraw from once <b>{ bigintToDecimalStringWithUnknown(requiredBalance.deepValue, 18n, 2) } REP</b> is reached. Micah commits (via a gentleman's agreement) to use all of the REP to fund an Augur v2 fork. The REP will be lost. Please find Micah's more detailed explanation at <a style = { { display: 'inline-flex', alignItems: 'flex-end' } } target = '_blank' rel = 'noopener noreferrer' href = { '/blog.html' }>
```

Comment:
```suggestion
						Deposit REP to REP Crowdsourcer that Micah can withdraw from once <b>{ bigintToDecimalStringWithUnknown(requiredBalance.deepValue, 18n, 2) } REP</b> is reached. Micah commits (via a gentleman's agreement) to use all of the REP to fund an Augur v2 fork. The REP will be lost. Please find Micah's more detailed explanation at <a style = { { display: 'inline-flex', alignItems: 'flex-end' } } target = '_blank' rel = 'noopener noreferrer' href = { './blog.html' }>
```

Reply:
This was wrong before too.  Always use relative paths, because you don't know if the site will be hosted at a subpath.